VAMobile/src/components/FormWrapper/FormFields/Picker/VAModalPicker.test.tsx

```diff
@@ -107,33 +103,31 @@ context('VAModalPicker', () => {
 
   describe('when labelKey exists', () => {
     it('should render a textview for the label', async () => {
-      const textViews = testInstance.findAllByType(TextView)
-      expect(textViews[7].props.children).toEqual(['Number', ' ', ''])
-      expect(textViews.length).toEqual(9)
+      expect(component.getByTestId('Number picker')).toBeTruthy()
+      expect(screen.getAllByText('Number').length).toBeGreaterThan(0)
     })
   })
 
   describe('when there is helper text', () => {
     it('should display it', async () => {
       await initializeTestInstance('js', 'label', 'back.a11yHint')
-      expect(testInstance.findAllByType(TextView)[8].props.children).toEqual('Navigates to the previous page')
+      expect(screen.getByText('Navigates to the previous page')).toBeTruthy()
+      expect(testInstance.findByProps({ children: 'Navigates to the previous page' })).toBeTruthy()
     })
   })
 
   describe('when there is an error', () => {
     it('should display it', async () => {
       await initializeTestInstance('email', 'label', '', 'ERROR')
-      const allTextViews = testInstance.findAllByType(TextView)
-      expect(allTextViews[allTextViews.length - 2].props.children).toEqual('ERROR')
+      expect(screen.getByText('ERROR')).toBeTruthy()
+      expect(testInstance.findByProps({ children: 'ERROR' })).toBeTruthy()
     })
   })
 
   describe('when isRequiredField is true', () => {
     it('should display (Required)', async () => {
```

Comment:
All the `async()`s can be removed from all the tests

Reply:
Updated.